spec/support/backend.rb

```diff
@@ -0,0 +1,37 @@
+module Backend
+  def self.from_name(name)
+    puts "Backend: #{name}"
+    case name
+    when 'active_record'
+      ActiveRecordBackend.new
+    when 'mongoid'
+      MongoidBackend.new
+    else
+      raise "Unknown backend #{name}, don't know how to run specs for that backend"
```

Comment:
Can we default to Active Record here? So if not environment variable is set then we assume that the tests are run with Active Record.